### ISSUE TITLE
F11 doesn't do anything

### ISSUE THREAD

**Issue by Shteiny** [SuiMachine/Deadly-Premonition-2---Sui-s-hack#19]
So I downloaded and installed the latest 0.6 version of melonloader onto the exe for dp2.
then I extracted the files from the 120.7z and copy pasted them into the game directory.
I have a controller that connects to and works with the game.
I launch the game everything seems right, i.e. the console opens with all the text and stuff and then the game starts.
but when I press F11 nothing happens.

**Comment by Shteiny** [comment]
Here is the last line of the console. I don't know what it means though.

![image](https://github.com/SuiMachine/Deadly-Premonition-2---Sui-s-hack/assets/75817604/82eaeafe-6faa-4832-ab1a-4d9c6c698a25)

**Comment by SuiMachine** [comment]
https://github.com/SuiMachine/Deadly-Premonition-2---Sui-s-hack/releases/tag/1.2.1

Apparently MelonLoader 0.6.2 or HarmonyX changed something with definition for TryGetSettings from the post process profile. I reworked it now.

**Comment by Nemesis3325** [comment]
I'm having the same issue. Even with downloading the newer release.

**Comment by KainXVIII** [comment]
I can confirm, mod is not working
![image](https://github.com/user-attachments/assets/9d155373-f26a-40ff-a6d7-947d6ef7b2b2)

**Comment by KainXVIII** [comment]
Temporary fix - install mod with MelonLoader version 0.6.2 (uncheck latest)

**Comment by SuiMachine** [comment]
I should just rewrite this to BepInEx, so I can ship everything in one package - am really annoyed with MelonLoader at this point 😞

**Comment by KainXVIII** [comment]
> I should just rewrite this to BepInEx, so I can ship everything in one package - am really annoyed with MelonLoader at this point 😞

Thank you in advance! (also maybe its possible to replace ALL button prompts to Playstation ones? Like start/select, rb/rt are still from xbox, which is a little annoying 😋)

**Comment by SuiMachine** [comment]
> > I should just rewrite this to BepInEx, so I can ship everything in one package - am really annoyed with MelonLoader at this point 😞
> 
> Thank you in advance! (also maybe its possible to replace ALL button prompts to Playstation ones? Like start/select, rb/rt are still from xbox, which is a little annoying 😋)

Yeah. I hope to have a look at it hopefully within the next month, cause that's been irritating me long enough, but want to close other things first.

**Comment by SuiMachine** [comment]
I have started reworking it to BepInEx. Had some issues figuring stuff out, but I am hoping to have a version for testing on the weekend.

**Comment by SuiMachine** [comment]
https://github.com/SuiMachine/Deadly-Premonition-2---Sui-s-hack/releases/tag/2.0.0RC1 - there is probably some things I have missed, but that entire thing is just so big it's hard to keep track of all the features

**Comment by KainXVIII** [comment]
> https://github.com/SuiMachine/Deadly-Premonition-2---Sui-s-hack/releases/tag/2.0.0RC1 - there is probably some things I have missed, but that entire thing is just so big it's hard to keep track of all the features

Thanks! Downloading now.

**Comment by SuiMachine** [comment]
If anything, I am a bit worried - since BepInEx with Il2cpp interop handles Coroutines differently that they might not be getting stopped correctly. Also had an issue with game objects being destroyed, but I think that should be fixed now.

**Comment by SuiMachine** [comment]
> Works fine, maybe only HBAO preset setting is buggy - because its sometimes works, sometimes don't do anything (but at least its doesn't crash like with previous melon loader version). PS - i think i understand now, HBAO settings works only after loading save file in current location but after you enter another one its doesn't work anymore (for example after you exit your hotel to the city map level).

Gotcha. Will see if that's one of the things I have missed.

**Comment by KainXVIII** [comment]
Works fine, maybe only HBAO preset setting is buggy - because its sometimes works, sometimes don't do anything (but at least its doesn't crash like with previous melon loeader version)

**Comment by SuiMachine** [comment]
Seems like some post-process layer references were being left in the list of post-process layers, despite the objects being destroyed, which would cause settings not to be applied from GUI after switching levels.

According to logs however, HBAO settings should be applied correctly regardless, just GUI didn't work.

https://github.com/SuiMachine/Deadly-Premonition-2---Sui-s-hack/releases/tag/2.0.0RC2
